spec/features/adding_peeps_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature 'Viewing peeps' do
+  scenario 'viewing related timestamp' do
```

Comment:
It seems like this test is just for viewing the peeps rather than timestamps, as the expected outcome is just a lil peep peep. I'm guessing you were halfway through adapting an existing test?

Reply:
Nice observation @Amy-O ! I agree, right now this test is not testing what it says it does.